owasm/example/bitcoin_info/src/logic.rs

```diff
@@ -0,0 +1,141 @@
+use owasm::ext::bitcoin::{block_count, block_hash};
+use owasm::{decl_data, decl_params, decl_result};
+
+decl_params! {
+    pub struct Parameter {
+        pub block_height: u64,
+    }
+}
+
+decl_data! {
+    pub struct Data {
+        pub block_hash: String = |params: &Parameter| block_hash::Info::new(params.block_height),
+        pub block_count: u64 = |_: &Parameter| block_count::Info::new(),
+    }
+}
+
+decl_result! {
+    pub struct Result {
+        pub block_hash: String,
```

Comment:
We have string as output how our solidity library need to support this?

Reply:
I will replace this thing by 4 of u64

Reply:
done